packages/gestalt-datepicker/src/DatePicker.jsdom.test.js

```diff
@@ -1,8 +1,20 @@
 // @flow strict-local
 import { useState } from 'react';
 import { render, screen, fireEvent } from '@testing-library/react';
+import { I18nProvider } from 'gestalt';
 import DatePicker from './DatePicker.js';
 
+const translations = {
```

Comment:
This is a bummer, but hopefully there aren't a lot of changes needed in the future here